lib/mincer/engines/sass_engine.js

```diff
@@ -65,11 +65,16 @@ function sassError(ctx /*, options*/) {
 // Render data
 SassEngine.prototype.evaluate = function (context/*, locals*/) {
   try {
-    this.data = sass.renderSync({
+    var result = sass.renderSync({
       data:         this.data,
       includePaths: [ path.dirname(this.file) ].concat(context.environment.paths),
       indentedSyntax: /^.*\.sass$/.test(this.file)
     });
+    if (result.css) {
+      this.data = result.css;
+    } else {
+      this.data = result;
+    }
```

Comment:
After reading the code in coffee_engine.js, adding source map support looks like adding `this.map = result.map;`

I've not tried this yet.